src/static/css/video-player.less

```diff
@@ -0,0 +1,221 @@
+
+/* ==========================================================================
+   cfgov-refresh
+   video-player
+   ========================================================================== */
+
+/* topdoc
+  name: Video Player
+  family: cfgov-video-player
+  patterns:
+      notes:
+        - "For the markup please see macros/video-player.html."
+  tags:
+    - cfgov-video-player
+*/
+
+// Variable used to set the size of the play button
+@play-btn-size__px: 73px;
+
+// Variable used to help center the video container
+@share-width__px: 130px;
+
+// Variables used for gutters
+@gutter__em: unit(@grid_gutter-width  / @base-font-size-px, em);
+@gutter-half__em: @gutter__em / 2;
+
+.video-player {
+    overflow: hidden;
+    background: @white;
+}
+
+.video-player.video-playing {
+    background-color: #000;
+
+    .hide-on_video-playing {
+        display: none;
+    }
+
+    .show-on_video-playing {
+        display: block;
+    }
+
+    .respond-to-max(@mobile-max, {
+        margin-top: 0;
+    });
+}
+
+// Video image container related styles
+
+.video-player_image-container {
+    position: relative;
+    background-color: #000;
+    text-align: center;
+
+    .respond-to-max(@mobile-max, {
+        .u-flexible-container();
+    });
+}
+
+.video-player_play-btn {
+    position: absolute;
+    height: @play-btn-size__px;
+    width: @play-btn-size__px;
+    background: url('/static/img/video-play-btn.png') no-repeat;
```

Comment:
Possibly the image path could be moved to a variable. Will this be bubbled up to CF? Then it would likely make sense to have a variable that could be overridden in cf-theme-overrides, otherwise it's less of a concern.